app/models/user.rb

```diff
@@ -144,6 +144,21 @@ def can_create_patient_laboratories?
     role?(Roles::PUBLIC_HEALTH) || role?(Roles::CONTACT_TRACER) || role?(Roles::PUBLIC_HEALTH_ENROLLER) || role?(Roles::SUPER_USER)
   end
 
+  # Can this user view Patient lab results?
+  def can_view_patient_dosages?
+    role?(Roles::PUBLIC_HEALTH) || role?(Roles::CONTACT_TRACER) || role?(Roles::PUBLIC_HEALTH_ENROLLER) || role?(Roles::SUPER_USER)
+  end
+
+  # Can this user edit Patient lab results?
+  def can_edit_patient_dosages?
+    role?(Roles::PUBLIC_HEALTH) || role?(Roles::CONTACT_TRACER) || role?(Roles::PUBLIC_HEALTH_ENROLLER) || role?(Roles::SUPER_USER)
+  end
+
+  # Can this user create Patient lab results?
+  def can_create_patient_dosages?
+    role?(Roles::PUBLIC_HEALTH) || role?(Roles::CONTACT_TRACER) || role?(Roles::PUBLIC_HEALTH_ENROLLER) || role?(Roles::SUPER_USER)
+  end
+
```

Comment:
@kevstoy @ChristineDuong What role(s) will the nurses be using? And will anyone beside nurses in those role(s) be creating/editing/viewing dosages?